corehq/ex-submodules/casexml/apps/case/tests/data/multicase/case_in_repeats.xml

```diff
@@ -28,8 +28,7 @@
 		<outcome>1</outcome>
 		<sex>1</sex>
 		<breastfeeding>2</breastfeeding>
-		<case>
-			<case_id>32337c03-2b73-497f-ab18-2b587f8c21df</case_id>
+		<case case_id="32337c03-2b73-497f-ab18-2b587f8c21df" xmlns="http://commcarehq.org/case/transaction/v2">
```

Comment:
Since v1 case XML isn't supported anymore we can also remove the v1 parsers. More code to kill :smiling_imp:

Reply:
I also noticed that we still have support for V1 restore. Should we remove that too?

Reply:
I would imagine we can but might need to check some analytics first